### PR TITLE
[App] moved common components to styled

### PR DESCRIPTION
**Pull request by Enjoy2Live**

### CODE REVIEW

File: src/components/common/Card/StyledCard.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      justify-content: space-between;
+    `}
+  && {
+    flex-direction: ${({ flexDirection }) =>
+      flexDirection ? flexDirection : "column"};
+  }
+  align-items: center;
+  text-align: ${({ textAlignment }) =>
+    textAlignment ? textAlignment : "left"};
+`;
```
Comment (NateD247) [inline src/components/common/Card/StyledCard.js:34]: Something here is a little off when ran on a local machine. On the home page, large card should have a flex direction of row, but it turns to flex direction of column. I am assuming it has something to do with line 27, but I am unsure. When you have time, run this on your local machine and take a look at the large card that says "Development Labs."

**Comment by Enjoy2Live** [comment]
> Overall, most of the changes check out! Many thanks for refactoring everything towards styled components. Please take a quick look at my comment on lines 16 through 34 about the large card. Thank you so much!

My bad for not checking all the instances of the card. it should be an easy fix!

**Comment by Enjoy2Live** [comment]
@NateD247 apparently you guys weren't adding the needed prop to the organization card so the styled components code was working just fine btw.

**Review by NateD247 — APPROVED** [review]
Thank you so much for refactoring the base and getting our large card layouts to display perfectly! 😄

**Pull request merged**